### ISSUE TITLE
Not able to generate personal API key as there is no option under My settings

### ISSUE THREAD

**Issue by Gsh-ops** [bitnami/bitnami-docker-testlink#108]
Not able to integrate Testlink with Selenium - Need API key and it is not showing any options under My setttings.

**Comment by dani8art** [comment]
Hi @Gsh-ops, thank you for opening this issue. 

Can you share with us more information on how are you deploying Testlink so that we can try to reproduce the issue in our environment?

**Comment by Gsh-ops** [comment]
Hello Dani8art, Thank you for the quick response.

We have followed stepts on below link -

https://github.com/bitnami/bitnami-docker-testlink#how-to-use-this-image

Used docker file to install and logged in as admin and clicked on the My settings.

**Comment by dani8art** [comment]
Hi @Gsh-ops I've followed the steps in the guide and I was not able to reproduce the issue. 

After `docker-compose up -d` I've gone to `My settings` and got this

![Screenshot 2020-09-22 at 10 10 50](https://user-images.githubusercontent.com/4376304/93858110-ed631900-fcbb-11ea-9d1a-165839226adb.png)

It was empty at the beginning but I clicked in `Generate a new key` and it showed up. 

Are you using docker-compose or docker command directly?
